### ISSUE TITLE
Use syntax supported by Nextflow language support

### ISSUE THREAD

**Issue by Jakob37** [genomic-medicine-sweden/tomte#189]
### Description of feature

As you might have seen, Nextflow has recently introduced a proper language server!

Meaning that developing in Nextflow DSL-2 gives proper editor support with error highlighting, auto-completion, quick navigation through references and so on. A huge difference for the dev experience :D

https://seqera.io/blog/modernizing-nextflow-developer-experience/

It can also visualize how processes / workflows connect. Here is the Tomte workflow illustrated by the VSCode plugin:

![tomte_dependencies](https://github.com/user-attachments/assets/7bbd6c30-c2c8-40aa-9e7b-5dfbe9db1f79)

Tomte is currently using some syntax not supported by this plugin, such as the `switch` statement and `for` loops. Not a lot, I just needed to touch a handful places to get it error free and working.

Would it make sense to adapt the Tomte syntax to align with the syntax expected here?

**Comment by jemten** [comment]
Good point! The addition of the nextflow language server is so nice and we should try to adhere to the standards. I think that @Lucpen also opened an issue to remove the `switch` logic from the pipeline, https://github.com/genomic-medicine-sweden/tomte/issues/183.
